src/index.njk

```diff
@@ -85,7 +89,7 @@ sitemapPriority: 1.0
                 <div class="w-40 m-auto mb-4">
                     {% image "./images/pictograms/book-a-demo.png", "Icon representing a booked demo.", [160] %}
                 </div>
-                <h5 class="flex justify-center">Talk to an Expert</h5>
+                <h5 class="flex justify-center">Learn More</h5>
                 <label class="font-light mt-2 block">The best way to learn about FlowFuse is to book a demo! We’d love to show you how FlowFuse can transform your business.</label>
                 <a class="mt-4 ff-btn ff-btn--secondary uppercase inline-block" href="/book-demo/" onclick="capture('cta-talk-to-expert', {'position': 'secondary'})">Book a Demo</a>
```

Comment:
@Yndira-FlowForge Can we analyze this change in Posthog? I've updated the CTA tag, so you should be able to validate it's impact